multiapps-controller-persistence/src/main/resources/org/cloudfoundry/multiapps/controller/persistence/db/changelog/db-changelog-1.120.0-core.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+
+    <changeSet author="sap.com" id="alter_operation_table_add_not_null_constraint_current_state">
+        <addNotNullConstraint tableName="operation"
+                              columnName="current_state"
+                              columnDataType="VARCHAR(50)"/>
```

Comment:
Is it necessary to specify data type again?

Reply:
It is not necessary for postgresql dialect, so I will remove it.